.concourse/post-publish.yaml

```diff
@@ -0,0 +1,346 @@
+
+resource_types:
+- name: pull-request
+  type: docker-image
+  source:
+    repository: teliaoss/github-pr-resource
+- name: github-status
+  type: docker-image
+  source:
+    repository: resource/github-status
+    tag: release
+resources:
+  - name: kubecf
+    type: git
+    source:
+      branch: master
+      uri: https://github.com/SUSE/kubecf
+  - name: catapult
+    type: git
+    source:
```

Comment:
To make this pipeline more deterministic, we need to specify a version of catapult here. Ideally, a commit hash.

Reply:
Solved in https://github.com/SUSE/kubecf/pull/360/commits/3d79327ffaaf087d92f170453272d275c5b9e3ad